graphql_compiler/tests/integration_tests/integration_test_helpers.py

```diff
@@ -16,15 +27,15 @@ def sort_db_results(results):
     Returns:
         List[Dict], sorted DB results.
     """
-    sort_order = []
+    sort_order: List[str] = []
     if len(results) > 0:
         sort_order = sorted(six.iterkeys(results[0]))
 
-    def sort_key(result):
+    def sort_key(result: Dict[str, Any]) -> Tuple[Tuple[bool, Any], ...]:
```

Comment:
I've gone down a bit of a rabbithole learning about `...` and I'm still not entirely sure what it means in this context. Would you mind explaining?

Reply:
This is a special case explained in https://github.com/python/mypy/issues/184

`Tuple[T, ...]` means an arbitrary-length tuple of `T` elements -- the `tuple()` analogue of `List[T]`. This is different than `Tuple[T]` which is a tuple of *one* `T` element.